### PR TITLE
Whodata 3.1+ & Alpine Linux support

### PR DESCRIPTION
**Pull request by ncvicchi**
## Description

Closes [19585](https://github.com/wazuh/wazuh/issues/19585), [6833](https://github.com/wazuh/wazuh-documentation/issues/6833), and [19412](https://github.com/wazuh/wazuh/issues/19412)

It has been added the procedure to allow Whodata to work on Alpine Linux (tested on 3.18.4 and 3.18.5) as well as a note to fix Whodata on newer Linux versions, where Auditd 3.1.1 or higher versions are used.

image

### Docs building
- [X] Compiles without warnings.
### Code formatting and web optimization
- [X] Uses three spaces indentation.
- [X] Adds or updates meta descriptions accordingly.
- [X] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [X] Uses present tense, active voice, and semi-formal registry.
- [X] Uses short, simple sentences.
- [X] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 1]
```diff
@@ -42,6 +42,37 @@ For Debian based systems, use the following command:
...
+      # apk add audit=3.1.1-r0
+      # rc-update add auditd default
+      # cp /usr/sbin/audisp-af_unix /sbin/audisp-af_unix
+
+
+.. note::
+
+      `Audit 3.1.1 changelog <https://people.redhat.com/sgrubb/audit/ChangeLog>`_. indicates that the audispd af_unix plugin was moved to a standalone program. For whodata to successfully connect to the Audit daemon, it is necessary to install this plugin.
+
+      In Red Hat based systems, use the following command to install the audspd af_unix plugin:
```
Comment (mjcr99) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:58]:
```suggestion
      In Red Hat based systems, use the following command to install the audispd af_unix plugin:
```

File: source/user-manual/capabilities/file-integrity/advanced-settings.rst [hunk 2]
```diff
@@ -42,6 +42,37 @@ For Debian based systems, use the following command:
...
+
+         .. code-block:: console
+
+            # yum install audispd-plugins
+
+      For Debian based systems, use the following command to install the audspd af_unix plugin:
```
Comment (mjcr99) [inline source/user-manual/capabilities/file-integrity/advanced-settings.rst:64]:
```suggestion
      For Debian based systems, use the following command to install the audispd af_unix plugin:
```

**Review by mjcr99 — CHANGES_REQUESTED** [review]
Review plugin name

**Review by mjcr99 — APPROVED** [review]
LGTM

**Review by javimed — CHANGES_REQUESTED** [review]
I would rather have it in this way using tabs. And why `service auditd restart`? Isn't `systemctl restart service` preferred?

imagen

imagen

imagen

I can commit this suggestions is accepted.

**Comment by ncvicchi** [comment]
> I would rather have it in this way using tabs. And why `service auditd restart`? Isn't `systemctl restart service` preferred?
> 
> imagen
> 
> imagen
> 
> imagen
> 
> I can commit this suggestions if accepted.

I just kept the style that was already there. I looks much better as you propose, so please commit it.

**Review by javimed — APPROVED** [review]

**Pull request merged**